support-lambdas/acquisition-events-api/src/test/scala/com/gu/acquisitionEventsApi/LambdaSpec.scala

```diff
@@ -44,10 +45,28 @@ class LambdaSpec extends AnyFlatSpec with Matchers {
     queryParameters = List(QueryParameter("name1", "value1"), QueryParameter("name2", "value2")),
     platform = None,
   )
+  // Create a new instance of ProxyRequestContext
+  val requestContext = new ProxyRequestContext()
+    .withAccountId("my-account-id")
+    .withStage("dev")
+    .withResourceId("my-resource-id")
+    .withRequestId("my-request-id")
+    .withHttpMethod("GET")
+    .withApiId("my-api-id")
+    .withPath("/my-resource-path")
+
+  // Set the authorizer map
+  val authorizer = None
+
+  // Set the RequestIdentity
+  val identity = None
 
   it should "run locally" in {
     val request = new APIGatewayProxyRequestEvent()
     request.setBody(acquisition.asJson.noSpaces)
+    request.setRequestContext(
+      requestContext,
```

Comment:
I have updated the test as  we  are currently adding request Id as well to the logs